pageserver/src/layered_repository/metadata.rs

```diff
@@ -16,10 +16,14 @@ use zenith_utils::{
 };
 
 use crate::{
```

Comment:
rustfmt complains that 
```suggestion
use crate::PageServerConf;
```

should be there now.